test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/Microsoft.AspNetCore.SignalR.Client.FunctionalTests.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitVersion)" />
     <PackageReference Include="xunit" Version="$(XunitVersion)" />
     <PackageReference Include="System.Reactive.Linq" Version="$(RxVersion)" />
+    <PackageReference Include="Microsoft.CSharp" Version="$(CoreFxVersion)" />
```

Comment:
Ok, if it's only for desktop, let's put it in a conditional property group then:

```xml
<PropertyGroup Condition="'$(TargetFramework)' == 'net46'">
    <PackageReference Include="Microsoft.CSharp" Version="$(CoreFxVersion)" />
</PropertyGroup>
```